client/components/feed.tsx

```diff
@@ -19,28 +18,17 @@ const GET_ORGANISATIONS = gql`
     }
 `;
 
-const Feed = () => {
-
-    const [searchText, setSearchText] = useState('');
+export function Feed() {
 
     const { loading, error, data } = useQuery(GET_ORGANISATIONS);
 
     if (loading) return "Loading...";
   
-    if (error) return `Error! ${error.message}`;
+    if (error) return `${error.message}`;
```

Comment:
Curious to why the component returns a string here, is it literally just showing as raw text on the client if there's an error?

Reply:
OK I've handled it differently, lmk if this is a good way.

Reply:
When I re-ran my tests, this one failed because my test looks for the error message to be printed to the client, [as described in this example](https://www.apollographql.com/docs/react/development-testing/testing/#network-errors).
You'll see that I now `expect` the hard-coded error message, but do not check for the error that I mocked in the test. Is that OK? Should I be checking to find the mocked error?

Reply:
Oh I meant that it's a bit weird for a React component to return pure text instead of HTML.

I would expect something like 
```
return (<div><blah><blah></div>)
```
including your error message on the HTML instead of just the error message itself. 

Remember this return value is getting rendered, so essentially what you have now is rendering as:

```
 </AppBar>
      <SearchBar/>
      error blah blah
    </>
```
on errors, which is kind of shit HTML.

It would be nicer to be something like:

```
 </AppBar>
      <SearchBar/>
      <div>
         <span style='color:red'>Error blah blah</span>
      </div
    </>
```

so you can style it, etc.